DatadogLogs/Sources/Feature/MessageReceivers.swift

```diff
@@ -94,33 +94,11 @@ internal struct LogMessageReceiver: FeatureMessageReceiver {
 internal struct CrashLogReceiver: FeatureMessageReceiver {
     private struct Crash: Decodable {
         /// The crash report.
-        let report: CrashReport
+        let report: DDCrashReport
         /// The crash context
         let context: CrashContext
     }
 
-    private struct CrashReport: Decodable {
```

Comment:
Yes 👍. This was the `CrashReport` structure sent from `DatadogCrashReporting` to `DatadogLogs` over message bus. Because in https://github.com/DataDog/dd-sdk-ios/pull/1687 we moved `DDCrashReport` to `DatadogInternal` we no longer need to encode and decode it over the bus.

In other words, the `DatadogCrashReporting.CrashReportSender` sends `DatadogInternal.DDCrashReport`, so here in `DatadogLogs` we can receive exactly the same type. This PR also adds `PassthroughCodable` conformance to `DDCrashReport`, so we can bypass serialization when passing it over message bus - which brings perf improvement when compared to decoding removed struct.